docs/sdks/server-sdks/dotnet/bandits.mdx

```diff
@@ -0,0 +1,209 @@
+        ["from"] = "germany"
+    }
+};
+
+var result = eppoClient.GetBanditAction(
+    "flag-with-shoe-bandit",
+    "user123",
+    subjectAttributes,
+    actions,
+    "default");
+
+if (result.Action != null)
+{
+    // Follow the Bandit action
+    RenderShoeAd(result.Action);
+} else {
+    // User was not selected for a Bandit.
+    // A variation is still assigned.
+    RenderDefaultShoeAd(result.Variation);
+}
+```
+
+### `GetBanditAction` Overloads
+
+There are several overloads of the `GetBanditAction()` method available:
+
+For a simple list of actions without attributes:
+```csharp
+public BanditResult GetBanditAction(
+    string flagKey,
+    string subjectKey,
+    IDictionary<string, object?> subjectAttributes,
+    string[] actions,
+    string defaultValue)
+```
+
+For a simple list of actions without attributes, using a `ContextAttributes` subject:
+```csharp
+public BanditResult GetBanditAction(
+    string flagKey,
+    ContextAttributes subject,
+    string[] actions,
+    string defaultValue)
+```
+
+Using `ContextAttributes` objects for subject and actions:
+```csharp
+public BanditResult GetBanditAction(
+    string flagKey,
+    ContextAttributes subject,
+    IDictionary<string, ContextAttributes> actions,
+    string defaultValue)
+```
+
+### Subject Context
+
+The subject context contains contextual information about the subject that is independent of bandit actions.
+For example, the subject's age or country.
+
+The subject context can be provided as `IDictionary<string, object?>`, which will then assume anything that is a number is a numeric
+attribute, and everything else is a categorical attribute.
+
+You can also explicitly bucket the attribute types by providing the context as `ContextAttributes`. For example, you may have an attribute named `priority`, with 
+possible values `0`, `1`, and `2` that you want to be treated categorically rather than numeric. `ContextAttributes` have two nested sets of attributes:
+- `NumericAttributes` (IDictionary): A mapping of attribute names to their numeric values (e.g., `age: 30`)
+- `CategoricalAttributes` (IDictionary): A mapping of attribute names to their categorical values (e.g., `country: "US"`)
+
+```csharp
+// Using ContextAttributes for explicit categorization
+var subjectContext = new ContextAttributes("user123")
+{
+    NumericAttributes = new Dictionary<string, double>
+    {
+        ["age"] = 30,
+        ["daysActive"] = 45
+    },
+    CategoricalAttributes = new Dictionary<string, string>
+    {
+        ["country"] = "US",
+        ["priority"] = "1"  // Treating numeric value as categorical
+    }
+};
+```
+
+### Action Context
+
+The action context contains contextual information about each action. They can be provided as a mapping of attribute names 
+to their contexts. 
+
+Similar to subject context, action contexts can be provided as `IDictionary<string, object?>`--which will then assume anything that is a number is a numeric
```

Comment:
emdash here as well
(there are some markdown post-processors which can convert `--` to a proper emdash, which is maybe what you're doing here, but it doesn't look like netlify is doing such post processing)

Reply:
Yep. In Obsidian, I have an extension that does that and I muscle memoried on it.

I opted to rewrite the sentence again. With some additional time away from these docs, I think most of the places where I tried to em-dash usually indicate a complex sentence that I should just rewrite into a couple simpler sentences, an unordered list, or both.